lib/graphql-pundit.rb

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```

Comment:
@eugenk what do you think of this? Shall I rename it or disable the check for the file?